.travis.yml

```diff
@@ -4,6 +4,8 @@ sudo: false
 
 jdk: oraclejdk8
 
+dist : trusty
```

Comment:
@adityastic And we were trying so hard, different ways to fix the issue xD. wow.

Reply:
@jddeep haha yeah.